src/tests/stdlib/array_remove_at.ab

```diff
@@ -0,0 +1,25 @@
+import { remove_at } from "std/array"
+
+// Output
```

Comment:
Together with `array_extract()`, this unit test is currently failing. I have boiled down the code under test to the simplest possible version, which I will post on Discord, along with a plea for help.